client/package.json

```diff
@@ -37,5 +38,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "dotenv": "^10.0.0"
```

Comment:
we might need this in production too, i am not really sure.

Reply:
I think we would, but should we cross that bridge later? or try and figure it out now?